src/main/java/core/basesyntax/converter/DataConverterImpl.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax.converter;
+
+import core.basesyntax.model.FruitTransaction;
+import java.util.ArrayList;
+import java.util.List;
+
+public class DataConverterImpl implements DataConverter {
+    @Override
+    public List<FruitTransaction> covertToTransaction(List<String> data) {
```

Comment:
The method name 'covertToTransaction' seems to be a typo. Consider renaming it to 'convertToTransaction' to better reflect the intended action.